contrib/depends/packages/qt.mk

```diff
@@ -1,18 +1,18 @@
 PACKAGE=qt
-$(package)_version=5.7.1
```

Comment:
Please add /monero#6822 to the end of the commit message.

Reply:
Resolved.